session.py

```diff
@@ -3,7 +3,7 @@
 from viur.core.tasks import PeriodicTask, callDeferred
 from viur.core import db, utils
 from viur.core.config import conf
-import logging
+import logging, pickle
```

Comment:
Can you explain why we should use pickle here?
The datastore supports the primitive data types, Key- and Entity-objects, list and dict (key must be a str) as value.
What type of data you like to store which isn't supported?